scripts/run_benches_for_pallets.sh

```diff
@@ -19,17 +19,19 @@ pallets=(
 	pallet-web3-names
 	parachain-staking
 	public-credentials
+	pallet-asset-switch
```

Comment:
Can you pull this out real quick in a separate PR?

Reply:
fair point: https://github.com/KILTprotocol/kilt-node/pull/800